### ISSUE TITLE
Allow 'action commands' such as Hide(e) and Show(e) to be called from the LUA _INIT function

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#4907]
I have a small standalone test of 4 levels. Once level 2 is loaded, Hide(e) and Show(e) stop working for multiple different scripts. These scripts function fine in the "Standalone Test All Levels" from the Storyboard, however once it is built this issue appears.

**Comment by Kitakazi** [comment]
Ok for whatever reason after the first level of a standalone if you have Show(e) or Hide(e) in the _init part of a script it will not work. That is the script works fine but just not Show(e) or Hide(e) if it is inside the _init. This might be true for any command in the _init part of a script but I have only tested with scripts that use Show(e) or Hide(e).

**Comment by LeeBamberTGC** [comment]
@Kitakazi Best not to use too many 'action commands' in the INIT part, just init values and other states, and do the in-game stuff in your main as this is called after the engines own object preparations.  You can trigger you show and hide commands using a simple init flag set in _init and detected and set in _main

**Comment by Kitakazi** [comment]
Ah ok sorry I am a noob when it comes to scripting, although I am getting better slowly but surely. TBH I'm not sure why I even put those commands in the _init part of the script, I made the scripts a while ago, I'm not really sure why I did that. 

However I guess my question is then, is it normal that the scripts would work fine in the Standalone Test All Levels mode but then once the standalone is built the scripts only work on level 1? Or is the object preparations handled differently for the Standalone Test All Levels feature?
